test/integration/src/test/java/org/opentripplanner/OtpHttpClientTest.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import jakarta.ws.rs.core.UriBuilder;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.opentripplanner.framework.io.OtpHttpClient;
+import org.opentripplanner.framework.io.OtpHttpClientFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This integration test makes sure that Apache HTTP client still works with important hosts and
+ * HTTP servers.
+ */
+class OtpHttpClientTest {
```

Comment:
I think @t2gran suggested the postfix for integration tests could be IntegrationTest.

Reply:
We could use the (admittedly pretty dusty) integration test feature of maven (via the failsafe plugin) or use the tags to select just these tests, like we do with `@GeneratesDocumentation`. Do you have an opinion?